instance/jmsml_D_Base.xml

```diff
@@ -41,10 +41,286 @@
+        <Value Name="ST" Label="Sao Tome and Principe" LabelAlias="STP"/>
+        <Value Name="SA" Label="Saudi Arabia" LabelAlias="SAU"/>
+        <Value Name="SN" Label="Senegal" LabelAlias="SEN"/>
+        <Value Name="RS" Label="Serbia" LabelAlias="SRB"/>
+        <Value Name="SC" Label="Seychelles" LabelAlias="SYC"/>
+        <Value Name="SL" Label="Sierra Leone" LabelAlias="SLE"/>
+        <Value Name="SG" Label="Singapore" LabelAlias="SGP"/>
+        <Value Name="SX" Label="Sint Maarten" LabelAlias="SXM"/>
+        <Value Name="SK" Label="Slovakia" LabelAlias="SVK"/>
+        <Value Name="SI" Label="Slovenia" LabelAlias="SVN"/>
+        <Value Name="SB" Label="Solomon Islands" LabelAlias="SLB"/>
+        <Value Name="SO" Label="Somalia" LabelAlias="SOM"/>
+        <Value Name="ZA" Label="South Africa" LabelAlias="ZAF"/>
+        <Value Name="GS" Label="South Georgia and South Sandwich Islands" LabelAlias="SGS"/>
+        <Value Name="SS" Label="South Sudan" LabelAlias="SSD"/>
+        <Value Name="ES" Label="Spain" LabelAlias="ESP"/>
+        <Value Name="XS" Label="Spratly Islands" LabelAlias="XSP"/>
+        <Value Name="LK" Label="Sri Lanka" LabelAlias="LKA"/>
+        <Value Name="SD" Label="Sudan" LabelAlias="SDN"/>
+        <Value Name="SR" Label="Suriname" LabelAlias="SUR"/>
+        <Value Name="XR" Label="Svalbard" LabelAlias="XSV"/>
+        <Value Name="SZ" Label="Swaziland" LabelAlias="SWZ"/>
+        <Value Name="SE" Label="Sweden" LabelAlias="SWE"/>
+        <Value Name="CH" Label="Switzerland" LabelAlias="CHE"/>
+        <Value Name="SY" Label="Syria" LabelAlias="SYR"/>
+        <Value Name="TW" Label="Taiwan" LabelAlias="TWN"/>
+        <Value Name="TJ" Label="Tajikistan" LabelAlias="TJK"/>
+        <Value Name="TZ" Label="Tanzania" LabelAlias="TZA"/>
+        <Value Name="TH" Label="Thailand" LabelAlias="THA"/>
+        <Value Name="TL" Label="Timor-Leste" LabelAlias="TLS"/>
+        <Value Name="TG" Label="Togo" LabelAlias="TGO"/>
+        <Value Name="TK" Label="Tokelau" LabelAlias="TKL"/>
+        <Value Name="TO" Label="Tonga" LabelAlias="TON"/>
+        <Value Name="TT" Label="Trinidad and Tobago" LabelAlias="TTO"/>
+        <Value Name="XT" Label="Tromelin Island" LabelAlias="XTR"/>
+        <Value Name="TN" Label="Tunisia" LabelAlias="TUN"/>
+        <Value Name="TR" Label="Turkey" LabelAlias="TUR"/>
+        <Value Name="TM" Label="Turkmenistan" LabelAlias="TKM"/>
+        <Value Name="TC" Label="Turks and Caicos Islands" LabelAlias="TCA"/>
+        <Value Name="TV" Label="Tuvalu" LabelAlias="TUV"/>
+        <Value Name="UG" Label="Uganda" LabelAlias="UGA"/>
+        <Value Name="UA" Label="Ukraine" LabelAlias="UKR"/>
+        <Value Name="AE" Label="United Arab Emirates" LabelAlias="ARE"/>
+        <Value Name="GB" Label="United Kingdom" LabelAlias="GBR"/>
+        <Value Name="US" Label="United States" LabelAlias="USA"/>
+        <Value Name="A1" Label="Unknown" LabelAlias="AX1"/>
```

Comment:
I'm glad this "Unknown" entry is here, I had actually forgotten about that important one.